src/components/PriceBox/PriceBox.jsx

```diff
@@ -1,6 +1,20 @@
 import "./pricebox.css";
+import { useCart } from "../../contexts";
+
+const PriceBox = (totalPrice) => {
+
+    const { cartState } = useCart();
+    const cartCounter = cartState.cartItems.length;
+    const finalAmount = totalPrice.tPrice - totalPrice.dPrice + 99;
+    const savedAmount = totalPrice.tPrice - finalAmount;
+
+    // const {_id, price, discountedPrice, qty } = product;
+
+    console.log("totalPrice:", totalPrice);
+    // console.log("discounted totalPrice in Cart:", totalDiscountedPrice);
+    console.log("finalamount:", finalAmount);
+    
```

Comment:
You can later on remove this commented console logs

Reply:
done!